opentreemap/treemap/templates/treemap/partials/udf_edit_field.html

```diff
@@ -9,6 +9,7 @@
 </select>
 {% elif field.type == 'date' %}
 <input class="form-control" type="text" data-field-name="{{ field.name }}"
+       data-date-start-date="{{ datepicker_start_date|date:request.instance.short_date_format|datepicker_format }}"
```

Comment:
`assumeNearbyYear` makes a better guess, but we still run into the problem of the date picker guessing what the user means, and then not giving them the opportunity to correct it. I'll ask the team for some more feedback.